SPECS/sudo.spec

```diff
@@ -63,16 +63,13 @@ BuildRequires:  openssl-devel
 %{name}-logsrvd is a high-performance log server that accepts event and I/O logs from sudo.
 It can be used to implement centralized logging of sudo logs.
 
-%if "%{dist}" != ".xs8~2_1"
 %package        python-plugin
 Summary:        Python plugin for %{name}
 Requires:       %{name} = %{version}-%{release}
 BuildRequires:  python3-devel
 
-
```

Comment:
There's no need to delete this line which is present in the XS-8.3 version of the spec file.